README.md

```diff
@@ -39,8 +39,11 @@ model = resnet50()
 # Load a configuration file to specify compression
 nncf_config = NNCFConfig.from_json("resnet50_int8.json")
 
+train_loader = # Your dataloader
+loss_criterion = # Your criterion (optional)
```

Comment:
Shouldn't add these lines

Reply:
okay. I will fix it in the following commits